.github/workflows/images-updater-core.yml

```diff
@@ -28,6 +32,8 @@ jobs:
       - name: Push latest image
         run: |
           docker push "$UPDATER_CORE_IMAGE:latest"
+          cosign sign --yes $(cosign triangulate --type=digest "$UPDATER_CORE_IMAGE:latest")
```

Comment:
The tag below should also resolve to the same digest, so we only need to sign once.

Reply:
Using `latest` is a bit dangerous. I'd prefer if we could use a hash

Reply:
Don't worry, `cosign triangulate` prints the hash behind a tag.

```
$ cosign triangulate --help
Outputs the located cosign image reference. This is the location where cosign stores the specified artifact type.

$ cosign triangulate --type=digest ghcr.io/dependabot/dependabot-updater-core:latest
ghcr.io/dependabot/dependabot-updater-core@sha256:5401993f730399e809a7ae35156674acb39a7aece707fddb0b0427c938c381d6
```